lib/internal/validators.js

```diff
@@ -228,6 +339,13 @@ function validatePort(port, name = 'Port', allowZero = true) {
   return port | 0;
 }
 
+/**
+ * @callback validateAbortSignal
+ * @param {string=} signal
```

Comment:
Oh and here it should not be a string.
```suggestion
 * @param {*} signal
```

Reply:
Done and force pushed. Thanks!